### ISSUE TITLE
I'm having issues with this tutorial

### ISSUE THREAD

**Issue by kevinlinxc** [DavidColby/turbo-frame-searching-rails#1]

**Comment by kevinlinxc** [comment]
It was a silly mistake, please delete this issue.

**Comment by kevinlinxc** [comment]
New issue: 

this doesn't happen for me:
![image](https://user-images.githubusercontent.com/44324811/193977754-2d937aac-32c6-4034-afd3-fdd49081d3fb.png)

It just goes to the normal players/index.html.erb. It also renders layouts/application.html.erb first

**Comment by devaniljr** [comment]
@kevinlinxc did you solve it?

**Comment by kevinlinxc** [comment]
No, I never figured this out unfortunately. I ended up abandoning Rails for my project.

**Comment by devaniljr** [comment]
I figured out the issue (I believe). Just remove `turbo_action: "advance"` from the form.

**Comment by kevinlinxc** [comment]
Unfortunately I won't be able to confirm this as I'm too far out of the context.
